operator-pipeline-images/Dockerfile

```diff
@@ -28,6 +28,7 @@ RUN dnf update -y && \
     gh \
     gnupg2 \
     jq \
+    nodejs-npm \
```

Comment:
We don't need a npn for this at all. The static check should be written as a python code which means we only need extra python dependency.